docs/debugging-testing/testing/unit_testing.md

```diff
@@ -124,76 +120,41 @@ With the following steps, you can write a unit test. This example creates dummy
     #endif
     ```
 
-    **MyClass.cpp**
-
-    ```
-    #include "MyClass.h"
-    #include "OtherClass.h"
+1. Add a new mbed-headers-example interface library: 
 
-    namespace example {
-
-    int MyClass::myFunction() {
-        OtherClass o = OtherClass();
-        return o.otherFunction();
-    }
-
-    }
-    ```
-
-    **OtherClass.h**
+   **mbed-os/UNITTESTS/stubs/CMakeLists.txt**
 
+   ```
+   target_include_directories(mbed-headers-example
+    INTERFACE
+        ${MBED_PATH}/example
+        ${MBED_PATH}/example/include
+        ${MBED_PATH}/example/include/example
+    )
     ```
-    #ifndef OTHERCLASS_H_
-    #define OTHERCLASS_H_
-
-    namespace example {
-
-    class OtherClass {
-    public:
-        int otherFunction();
-    };
 
-    }
+1. Update the newly created `mbed-headers-example` library into exising target link library of `mbed-headers`.
```

Comment:
I do not understand what this sentence means.

Reply:
I have rephrased the sentence